.travis.yml

```diff
@@ -4,23 +4,25 @@ os:
   - linux
 
 dist:
-  - bionic
+  - focal
 
 compiler:
   - gcc
 
 addons:
   apt:
     sources:
-      - sourceline: "deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ bionic-updates main universe"
+      - sourceline: 'ppa:chris-lea/redis-server'
     packages:
-      - qemu-user
+    - redis-tools
```

Comment:
Why are redis tools and server included?

Reply:
Removed